conda/recipes/libcuml/meta.yaml

```diff
@@ -68,10 +68,12 @@ requirements:
     - libcusolver-dev
     - libcusparse-dev
     {% endif %}
+    - fmt {{ fmt_version }}
     - libcumlprims ={{ minor_version }}
     - libraft ={{ minor_version }}
     - libraft-headers ={{ minor_version }}
     - librmm ={{ minor_version }}
+    - spdlog {{ spdlog_version }}
```

Comment:
These are in the global list for the split recipe but also need to be in the `libcuml` package below in order to have run-exports on the appropriate packages. That would eliminate the need for test-time pinnings. I'm not sure which direction we want to go: do we add run-exported dependencies on these to "runtime" packages (these aren't "dev" packages) to enforce consistency, or use test-time pinnings to only ensure that we get compatible builds of librmm, etc. which are exporting these dependencies (and with which we need to avoid conflicts)?

Reply:
Ah ok! I'm not that familiar with the ways that split recipes are different from single-package ones, thank you for pointing this out.

I'm not sure precisely what you mean about those two choices... but I can say at least that I think `cuml` / `libcuml` should explicitly pin `fmt` and `spdlog`, given that it directly uses them: https://github.com/rapidsai/cuml/pull/6071#discussion_r1771772139 (instead of getting that pinning transitively from `librmm` or similar).

Reply:
We talked offline and decided to add these pinnings in the top-level host environment and the *build* environment created from the `common_build:` list in `dependencies.yaml`, but to not mess with run exports or adding explicit run dependencies on `fmt` / `spdlog`.